lib/openstax/rescue_from/logger.rb

```diff
@@ -0,0 +1,31 @@
+module OpenStax
+  module RescueFrom
+    class Logger
+      attr_reader :proxy
+
+      def initialize(proxy)
+        @proxy = proxy
+      end
+
+      def record_system_error!(prefix = "An exception occurred")
+        Rails.logger.error("#{prefix}: #{proxy.name} [#{proxy.error_id}] " +
```

Comment:
Nah, there was a whole discussion in Skype you missed when I was having initialization problems with the gem engine and `Rails.logger` was nil when configuration was being set, so we thought, do we even need to configure this? Not today, we don't.